src/Apps/Order2/Routes/Checkout/Components/DeliveryOptionsStep/Order2DeliveryOptionsStep.tsx

```diff
@@ -22,13 +22,13 @@ export const Order2DeliveryOptionsStep: React.FC<
 
   return (
     <Flex flexDirection="column" backgroundColor="mono0">
-      <Box p={2} data-testid="DeliveryOptionsStep">
+      <Box py={2} px={4} data-testid="DeliveryOptionsStep">
```

Comment:
I think we want to keep the mobile padding to 2, so I would suggest that we use the responsive props to set it for each breakpoint
something like this:
```tsx
<Box py={2} px={[2, 4]} data-testid="DeliveryOptionsStep">
```

also for the rest of the changes, for more info check out our [Palette doc](https://palette.artsy.net/guides/responsive/) about responsive props